src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpander.cs

```diff
@@ -26,6 +26,21 @@ public class LanguageViewLocationExpander : IViewLocationExpander
     {
         private const string ValueKey = "language";
 
+        public LanguageViewLocationExpander(
+            LanguageViewLocationExpanderOption option = LanguageViewLocationExpanderOption.Suffix)
```

Comment:
A boolean isn't very usable in this case as you are selecting between two distinct behaviors, rather than enabling/disabling one thing.